packit_service/models.py

```diff
@@ -169,6 +169,12 @@ class CoprBuild(Base):
     build_submitted_time = Column(DateTime, default=datetime.utcnow)
     build_start_time = Column(DateTime)
     build_finished_time = Column(DateTime)
+    # git url
+    https_url = Column(String)
```

Comment:
2 things:
1. I don't see any code which would ingest this
2. git web url is not directly related to a copr build, can we move that field to the GitProject model?

Could you also put an example data to the comment? we have so many URLs that it's so easy to get lost

Reply:
> I don't see any code which would ingest this

I saw a fixme in the code somewhere where they were creating this again from available data and it said to replace it with an field in psql so I created this field.

We can then change the code later.

Reply:
> > I don't see any code which would ingest this
> 
> I saw a fixme in the code somewhere where they were creating this again from available data and it said to replace it with an field in psql so I created this field.
> 
> We can then change the code later.

right, oki